rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_wind_plugin.h

```diff
@@ -135,6 +140,33 @@ class GazeboWindPlugin : public ModelPlugin {
   common::Time wind_gust_end_;
   common::Time wind_gust_start_;
 
+  /// \brief    Variables for custom wind field generation.
+  bool custom_static_wind_field_;
+  float min_x_;
+  float min_y_;
+  int n_x_;
+  int n_y_;
+  float res_x_;
+  float res_y_;
+  std::vector<float> vertical_spacing_factors_;
+  std::vector<float> bottom_z_;
+  std::vector<float> top_z_;
+  std::vector<float> u_;
+  std::vector<float> v_;
+  std::vector<float> w_;
+  
+  /// \brief  Reads wind data from a text file and saves it.
+  /// \param[in] custom_wind_field_path Path to the wind field from ~/.ros.
+  void ReadCustomWindField(std::string& custom_wind_field_path);
```

Comment:
What I meant was: Define all functions that do not need to modify member variables as `const`. Otherwise leave as is.